src/chunked.rs

```diff
@@ -143,11 +143,11 @@ impl<'a> ChunkedMessageId {
     fn random() -> ChunkedMessageId {
         let mut bytes = [0; 8];
 
-        for b in 0..8 {
-            bytes[b] = rand::random();
+        for byte in &mut bytes {
+            *byte = rand::random();
```

Comment:
`fill_bytes` from `rand` would be a better refactoring here

Reply:
done